entry/vexflow-core.ts

```diff
@@ -51,10 +51,10 @@ Flow.fetchMusicFont = async (fontName: string, fontModuleOrPath?: string | FontM
     const module = await import(/* webpackIgnore: true */ fontModuleOrPath);
 
     const g = globalObject();
-    const moduleCJS = g['VexFlowFont' + fontName];
-    if (typeof moduleCJS !== 'undefined') {
-      // CJS font modules will set a global variable named: VexFlowFontBravura | VexFlowFontGonville | VexFlowFontPetaluma | etc.
-      fontModule = moduleCJS;
+    const VexFlowFont = g['VexFlowFont'];
+    if (typeof VexFlowFont !== 'undefined' && typeof VexFlowFont[fontName] !== 'undefined') {
+      // CJS font modules will set a global variable named: VexFlowFont.Bravura | VexFlowFont.Gonville | VexFlowFont.Petaluma | etc.
+      fontModule = VexFlowFont[fontName];
```

Comment:
This is only for `vexflow-core.js` and dynamic font module loading. A CJS font module will set the `VexFlowFont.{FontName}` with the object containing the glyphs and metrics information (e.g., VexFlowFont.Bravura, or VexFlowFont.Leland if you are adding the Leland SMuFL font).